docs/examples.md

```diff
@@ -0,0 +1,363 @@
+    "neo4j"
+    "password"
+    {:log {:level :info}
+           :encryption :none}))
+~~~
+
+In the current version we support the following options:
+
+~~~
+:log :level [:all :error :warn :info :off] - defaults to :warn
+:encryption [:required :none] - defaults to :required"
+~~~
+
+## Disconnect from neo4j server
+
+~~~clojure
+(client/disconnect conn)
+~~~
+
+## Execute queries
+
+This section shows how to execute raw bolt queries against on a open connection
+
+### Execute query without parameters
+
+~~~clojure
+(client/execute! conn "MATCH (n:Person) RETURN n")
+~~~
+
+### Execute query with parameters
+
+~~~clojure
+(client/execute! conn "MATCH (n:Person {firstName: $first_name}) RETURN n" {:first_name "Neo"})
+~~~
+
+Notice that dashes is not allowed in parameter names
+
+## Node CRUD
+
+This section shows how to do basic CRUD operations on nodes through Neo4clj convenience functions.
+To learn more about the Clojure representation of a node please see our [representations](representations.md) page
+
+### Create a Node
+
+~~~clojure
+(client/create-node! conn {:ref-id "p"
+                           :labels [:person]
+                           :props {:first-name "Neo"
+                                   :last-name "Anderson"}})
+~~~
+
+### Find nodes in Neo4j
+
+The entry given to `find-nodes!` is a lookup representation. To learn more about the Clojure lookup representation see our [representations](representations.md) page
+
+~~~clojure
+(client/find-nodes! conn {:ref-id "p"
+                          :labels [:person]
+                          :props {:first-name "Neo"
+                                  :last-name "Anderson"}})
+~~~
+
+### Update and Delete Node
+
+See the section [Update and Delete Entity](#update-and-delete-neo4j-entity)
+
+
+## Relationship CRUD
+
+This section shows how to do basic CRUD operations on relationships through Neo4clj convenience functions.
+To learn more about the Clojure representation of a relationship please see our [representations](representations.md) page
+
+### Create a Relationship
+
+The keys `from` and `to` are Lookup representations. To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+~~~clojure
+(client/create-relationship! conn {:ref-id "p"
+                                   :type :employee
+                                   :from {:labels [:person] :props {:first-name "Neo"}}
+                                   :to {:id 12}
+                                   :props {:position "Developer"}})
+~~~
+
+### Find relationships in Neo4j
+
+To fetch relationships from the Neo4j database, please refer to the section [Get Graph](#get-graph)
+
+### Update and Delete Relationship
+
+See the section [Update and Delete Entity](#update-and-delete-neo4j-entity)
+
+
+## Update and Delete Neo4j Entity
+
+In this section you can find examples on how to update specific parts a Node or Relationship and how to delete them.
```

Comment:
I think an introduction to the term "entity" would be nice.

I was thinking something along the lines:
> Nodes and relationships are both "entities" and this section will describe how to update and delete them.